packages/terra-tag/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "terra-tag",
+  "main": "lib/Tag.js",
+  "version": "0.0.0",
+  "description": "The tag component is used for tagging and provides users a way to trigger actions in the UI. Each tag should have text. It can optionally display an icon along with the text.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-tag",
+    "Tag",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "peerDependencies": {
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "terra-base": "^2.0.0"
```

Comment:
Need to update this to `^3.0.0`. The 3.0.0 version relies on react 16, the 2.0.0 versions rely on react 15.

Reply:
Update it [here](https://github.com/cerner/terra-core/pull/1306/commits/8e3c99f0e06259ee8eb0b2b7e435c28cf0feca0c).